### PR TITLE
Make JSON-B and Jackson equals in the REST JSON guide

### PR DESCRIPTION
**Pull request by gsmet**
/cc @emmanuelbernard 

@geoand could you have a look? Thanks!

### CODE REVIEW

File: docs/src/main/asciidoc/rest-json-guide.adoc [hunk 1]
```diff
@@ -474,6 +478,12 @@ was served.
 Quarkus comes with a CORS filter. Read the link:undertow-reference.html#cors-filter[Undertow Reference Documentation] to learn
 how to use it.
 
+[WARNING]
+====
+If you want to use the `keycloak` Quarkus extension in your project, you should configure CORS using the `keycloak` extension
+configuration. See link:keycloak-guide#configuring-cors[the Keycloak guide] for more details.
+====
```
Comment (gsmet) [inline docs/src/main/asciidoc/rest-json-guide.adoc]: This is unrelated but it has been added in the Undertow guide but should also be there to redirect users directly to the Keycloak guide.

**Comment by geoand** [comment]
Sure thing, I'll check it in a few hours

**Review by geoand — APPROVED** [review]
Does exactly what is advertised IMHO :)

**Comment by gsmet** [comment]
Merging, then, thanks!

**Comment by geoand** [comment]
Don't mention it!

**Pull request merged**